src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.html

```diff
@@ -50,8 +50,13 @@ <h4>{{ 'ubs-tariffs-add-service.edit_service' | translate }}</h4>
           formControlName="description"
           id="description"
           placeholder="{{ 'ubs-tariffs-add-service.placeholder-text' | translate }}"
+          maxlength="255"
+          required
         >
         </textarea>
+        <div class="validation" *ngIf="getControl('description').invalid && getControl('description').touched">
+          {{ 'ubs-tariffs-add-service.error_service_field_message' | translate }}
```

Comment:
Use **app-ubs-input-error** for ubs error messages
For example we use it in **UBSAddAddressPopUpComponent**